docs/content/Checkbox.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Checkbox
+status: Alpha
+source: https://github.com/primer/react/blob/main/src/Checklist.tsx
+---
+
+import {ComponentChecklist} from '../src/component-checklist'
+
+`Checkbox` is a form component that applies Primer design language to a native HTML checkbox input element.
+
+## Default example
+
+The `Checkbox` component can be used in controlled and uncontrolled modes.
+
+```jsx live
+<>
+  <Box as="form" p={3} pt={0} sx={{display: 'flex', alignItems: 'center'}}>
+    <Checkbox id="default-checkbox" />
+    <Text as="label" htmlFor="default-checkbox" sx={{fontSize: 2, fontWeight: 'bold', marginLeft: 1}}>
+      Default checkbox
+    </Text>
+  </Box>
+  <Box as="form" p={3} sx={{display: 'flex', alignItems: 'center'}}>
+    <Checkbox id="always-checked-checkbox" checked={true} />
+    <Text as="label" htmlFor="always-checked-checkbox" sx={{fontSize: 2, fontWeight: 'bold', marginLeft: 1}}>
+      Always checked
+    </Text>
+  </Box>
+  <Box as="form" p={3} sx={{display: 'flex', alignItems: 'center'}}>
+    <Checkbox id="always-unchecked-checkbox" checked={false} />
+    <Text as="label" htmlFor="always-unchecked-checkbox" sx={{fontSize: 2, fontWeight: 'bold', marginLeft: 1}}>
+      Always unchecked
+    </Text>
+  </Box>
+
+  <Box as="form" p={3} sx={{display: 'flex', alignItems: 'center'}}>
+    <Checkbox id="inactive-checkbox" checked={true} disabled />
+    <Text as="label" htmlFor="inactive-checkbox" sx={{fontSize: 2, fontWeight: 'bold', marginLeft: 1}}>
+      Inactive
+    </Text>
+  </Box>
+</>
+```
+
+import { useState } from 'react'
+import { useMDXScope } from "gatsby-plugin-mdx/context";
+
+export const MyDynamicButton = () => {
+const [count, setCount] = useState(0)
+return (
+<button onClick={() => setCount(count + 1)}>Clicked {count} times!</button>
+)
+}
```

Comment:
What is this for?

Reply:
Wow.. thanks @colebemis... that's a hangover from some mdx testing. Removed now 👍